Library/Homebrew/os/mac/sdk.rb

```diff
@@ -0,0 +1,57 @@
+require "os/mac/version"
+
+module OS
+  module Mac
+    class SDK
+      attr_reader :version, :path
+
+      def initialize version, path
```

Comment:
I think we settled on parentheses around method arguments on method definition, haven't we? Could you run your code through `brew style` as it will complain about this and a few other minor things?